sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1733,6 +1733,19 @@ object functions {
    */
   def atan2(yValue: Double, xName: String): Column = atan2(yValue, Column(xName))
 
+  /**
+   * @param yValue coordinate on y-axis
+   * @param xValue coordinate on x-axis
+   * @return the <i>theta</i> component of the point
+   *         (<i>r</i>, <i>theta</i>)
+   *         in polar coordinates that corresponds to the point
+   *         (<i>x</i>, <i>y</i>) in Cartesian coordinates,
+   *         as if computed by `java.lang.Math.atan2`
+   * @group math_funcs
+   * @since 3.4.0
+   */
+  def atan2(yValue: Double, xValue: Double): Column = atan2(lit(yValue), lit(xValue))
```

Comment:
I think we should probably fix it in PySpark side instead of fixing it in Scala side (see the comments on the top at both functions.py and functions.scala)

Reply:
Sure, will do